operators/pkg/forge/containers.go

```diff
@@ -61,6 +61,10 @@ const (
 	AppMEMLimitsEnvName = "APP_MEM_LIMITS"
 	// PodNameEnvName -> name of the env variable containing the Pod Name.
 	PodNameEnvName = "POD_NAME"
+	// Name of the NFS volume
+	NFSVolumeName = "nfs-vol"
+	// Mount path for the NFS personal volume
+	NFSVolumeMountPath = "/media/MyDrive"
```

Comment:
Still some issues with naming here. Having `/mydrive` and `media/MyDrive` is likely to lead to confusion. To me, the feature you are introducing is the one that should be named mydrive (for consistency with the VMs), while the previous one should be changed to something else (e.g., `/media/persistent`). Also the variable names should reflect this. @QcFe WDYT?

Reply:
Names changed